pkg/serverless/protocol.go

```diff
@@ -335,6 +316,33 @@ func (l *LogsCollection) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// processMessage performs logic about metrics and tags on the message
+func enhanceMessage(message aws.LogMessage, arn string, lastRequestID string, functionName string, computeEnhancedMetrics bool, metricTags []string, metricsChan chan []metrics.MetricSample) {
```

Comment:
The docstring says `processMessage` but the function is called `enhanceMessage`...

I think `processMessage` is a better name. Thus function does other things besides enhancing the message -- it also sets the global request ID, cold start variable, among other things...

Reply:
✅